src/storage/models/storageManager.ts

```diff
@@ -0,0 +1,27 @@
+import { Logger } from '@map-colonies/js-logger';
+import { inject, injectable } from 'tsyringe';
+import config from 'config';
+import { SERVICES } from '../../common/constants';
+import { IStorageStatusResponse } from '../../common/interfaces';
+import { getStorageStatus } from '../../common/utils';
+
+@injectable()
+export class StorageManager {
+  private readonly gpkgsLocation: string;
+  
+  public constructor(
+    @inject(SERVICES.LOGGER) private readonly logger: Logger
+  ) {
+    this.gpkgsLocation = config.get<string>('gpkgsLocation');
+  }
+
+  public async getStorage(): Promise<IStorageStatusResponse> {
+    const storageStatus: IStorageStatusResponse = await getStorageStatus(this.gpkgsLocation);
+    this.logger.debug(storageStatus, `Current storage free and total space for gpkgs location`);
```

Comment:
its mutual Q to Asaf also, @asafMasa  if should added some warning log in case it return "zeros" about the status, becouse known problem that the libary storage not support some storage\volumes types that return always 0

Reply:
no need for warn log